pkg/config/config.go

```diff
@@ -37,37 +37,37 @@ var (
 )
 
 type ClusterConfig struct {
-	URL string `json:"url"`
-
-	ClusterCIDR          string `json:"clusterCIDR"`
-	ServiceCIDR          string `json:"serviceCIDR"`
-	ServiceNodePortRange string `json:"serviceNodePortRange"`
-	DNS                  string `json:"dns"`
-	Domain               string `json:"domain"`
-	MTU                  string `json:"mtu"`
+	// URL string `json:"url"`
+
+	// ClusterCIDR          string `json:"clusterCIDR"`
+	// ServiceCIDR          string `json:"serviceCIDR"`
+	// ServiceNodePortRange string `json:"serviceNodePortRange"`
+	// DNS string `json:"dns"`
+	// Domain               string `json:"domain"`
+	// MTU string `json:"mtu"`
```

Comment:
@zshi-redhat @mangelajo does MTU have to be user-configurable or can we auto-detect it somehow?

Reply:
@fzdarsky Until the path mtu is verified to work in ovnk, we need the mtu to be configurable.

Reply:
@benluddy @dgrisonnet MTU is stored on NetworkStatus.  You sure you don't want the entire config objects as opposed to just spec?

Reply:
Sometimes PMTU won't work because something on a remote network is broken and out of the customer's control. In this case, they need to be able to limit MTU locally to workaround issues.

Reply:
Using the NetworkStatus that is supposed to be read-only in OCP to specify a configuration in Microshift is not ideal and kind of confusing, I'd rather diverge from the existing config spec and do conversion later on than bringing the status.

Reply:
I agree with @dgrisonnet. MTU is an example of a way that MicroShift needs to be configurable that (right now) OCP does not.